### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by dytk2134**

### CODE REVIEW

File: remap-gff3/bin/remap-gff3.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def filter_not_exact_match(CrossMap_mapped_file, CrossMap_log_file, filtered_fil
                             if attribute_dict['ID'] in ID_set:
                                 logger.error('All features should have an unique ID. Please run the command with -tmp_ID.')
                                 sys.exit(0)
-                            else:
+                            elif not 'not exact match' in match_state:
```
Comment (hsiaoyi0504) [inline remap-gff3/bin/remap-gff3.py]: Although the syntax have the same effect of `elif not 'not exact match' in match_state`, it seems to me that using `if item not in itemlist` will be better. Similar syntax is explained in PEP8 https://www.python.org/dev/peps/pep-0008/.

Comment (dytk2134) [reply remap-gff3/bin/remap-gff3.py]: Why `if item not in itemlist` is better?

Comment (hsiaoyi0504) [reply remap-gff3/bin/remap-gff3.py]: The "better" means more readable here. See error code 713 in  pep8 python style guide checker.
http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes

**Comment by hsiaoyi0504** [comment]
@dytk2134 Sorry, I still didn't get it. Where are the **example1_CrossMap_filtered.gff3** and **example_file/example1_updated.gff3** ? I saw that it's in quick start part of the **README.md** but can't find them in the repo.

**Comment by hsiaoyi0504** [comment]
Fixed in #18.

**Pull request merged**